### PR TITLE
 Document export compiled dependencies into a different format

### PR DESCRIPTION
**Pull request by JavierMatosD**
The tutorial walks a user through installing, exporting, and consuming a library.

### CODE REVIEW

File: vcpkg/produce/export-compiled-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+vcpkg export fmt:x64-windows --zip
+```
+
+You should see something similar to the following:
+
+```console
+The following packages are already built and will be exported:
+    fmt:x64-windows
+  * vcpkg-cmake:x64-windows
+  * vcpkg-cmake-config:x64-windows
```
Comment (Thomas1664) [inline vcpkg/produce/export-compiled-dependencies.md:47]: Aren't host dependencies useless in this case?
As far as I know they're only supposed to be used during the build of a port but here the port is already built.

File: vcpkg/produce/export-compiled-dependencies.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+## 1 - Install `fmt` library
+
+Begin by opening your terminal and navigating to the directory where vcpkg is installed. Once there, install the `fmt` library for your target platform:
+
+```console
+vcpkg install fmt:x64-windows
+```
+
+Replace `x64-windows` with your specific target triplet if different.
```
Comment (vicroms) [inline vcpkg/produce/export-compiled-dependencies.md:31]:
```suggestion
Replace `x64-windows` with your target triplet.
```

File: vcpkg/produce/export-compiled-dependencies.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+## Prerequisites:
+
+- [Visual Studio](<https://visualstudio.microsoft.com/downloads/>) with C++ development workload and [CMake component](</cpp/build/cmake-projects-in-visual-studio#installation>)
+- [Git](<https://git-scm.com/downloads>)
+* vcpkg
+* A terminal
+
+## 1 - Install `fmt` library
+
+Begin by opening your terminal and navigating to the directory where vcpkg is installed. Once there, install the `fmt` library for your target platform:
```
Comment (vicroms) [inline vcpkg/produce/export-compiled-dependencies.md:25]:
```suggestion
Open a terminal and navigate to the directory where vcpkg is installed then install the `fmt` library for your target platform:
```

File: vcpkg/produce/export-compiled-dependencies.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+```console
+vcpkg install fmt:x64-windows
+```
+
+Replace `x64-windows` with your specific target triplet if different.
+
+## 2 - Export `fmt` library
+
+After installing `fmt`, you can export it by running:
```
Comment (vicroms) [inline vcpkg/produce/export-compiled-dependencies.md:35]:
```suggestion
After installing `fmt`, export it by running:
```

File: vcpkg/produce/export-compiled-dependencies.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+```console
+vcpkg export fmt:x64-windows --zip
+```
+
+You should see something similar to the following:
```
Comment (vicroms) [inline vcpkg/produce/export-compiled-dependencies.md:41]:
```suggestion
You should see output similar to the following:
```

File: vcpkg/produce/export-compiled-dependencies.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+Additional packages (*) need to be exported to complete this operation.
+Exporting vcpkg-cmake:x64-windows...
+Exporting vcpkg-cmake-config:x64-windows...
+Exporting fmt:x64-windows...
+Creating zip archive...
+Zip archive exported at: C:\dev\vcpkg\vcpkg-export-20231106-164550.zip
+To use exported libraries in CMake projects, add -DCMAKE_TOOLCHAIN_FILE=[...]/scripts/buildsystems/vcpkg.cmake to your CMake command line.
+```
+
+This command will create a zip archive containing all the necessary files to use the `fmt` library, including binaries and headers. For a list of all the supported formats check out the reference [documentation](../commands/export.md#formats).
```
Comment (vicroms) [inline vcpkg/produce/export-compiled-dependencies.md:57]:
```suggestion
The command creates a zip archive containing all the necessary files to use the `fmt` library, including binaries and headers. For a list of all the supported formats check out the [`export` command documentation](../commands/export.md#formats).
```

**Review by vicroms — CHANGES_REQUESTED** [review]

**Review by vicroms — APPROVED** [review]

**Pull request merged**